### DISCUSSION TITLE
Tutorial?

### DISCUSSION THREAD

**Discussion by eknuth1002** [TokisanGames/Terrain3D#273] [Questions] [answered]
I am looking at Terrain3D because I am not good with visual assets, like at all, and thought this might be a great way to make something even with little skill.  So, I am unfamilar with techniques for materials and such.  I mention this because I am not sure if I am using Terrain3d incorrectly or not in Godot 4.2.  I am getting the checkerboard pattern for a basic landscape (which I believe is default with no textures), but I cannot paint anything or get the NavMesh option to display at all.

Is there any form of tutorial planned outside the basics in the docs?  I am more the style of "do A then B then C to get a basic terrain" and help us asset dummies out.

**Comment by TokisanGames** [comment]
I'm editing a video which will be released along with the beta release by this weekend. If you can't find the navigation button, it's because that's in the beta version. The docs don't differentiate between versions right now.

If you're using the build with navigation, then you're using an experimental build. Using experimental builds comes with some technical responsibility of figuring things out by yourself.

**Reply by eknuth1002** [reply]
I don't believe I'm using experimental (using the nightly build for 4.2).  That's probably where I'm getting thrown off in the docs since there's currently no differentiation there, and I was trying to learn though that.  I will keep an eye out for the beta and video.  I am really looking forward to seeing this progress and how it's used in your game.  Thanks.

**Reply by TokisanGames** [reply, marked as answer]
Youtube tutorial. Part 2 will come in a few days.
https://youtu.be/oV8c9alXVwU
